ProcessLib/Output/Output.h

```diff
@@ -119,7 +122,7 @@ class Output
     ProcessData* findProcessData(Process const& process, const int process_id);
 
     //! Determines if there should be output at the given \c timestep or \c t.
-    bool shallDoOutput(unsigned timestep, double const t);
+    bool shallDoOutput(int timestep, double const t);
```

Comment:
Is there any reason the timestep isn't `const`?

Reply:
Two points: in the declaration the `const` can be omitted anyway (even recommended by google's style guide), and secondly it is also non-const in the function's definition where it is used as a local variable.